src/metrics.rs

```diff
@@ -38,13 +38,17 @@ async fn get_metrics() -> impl Responder {
 
     let mut buffer = Vec::new();
     if let Err(e) = encoder.encode(&prometheus::gather(), &mut buffer) {
-        eprintln!("could not encode metrics: {}", e);
+        tracing::error!(target: LOGGING_PREFIX, "could not encode metrics: {}", e);
```

Comment:
Do we now have no appearances of `println/eprintln` in the project?

Reply:
Yes, these are the only two instances where a `println/eprintln` is used.